README.MD

```
@@ -175,11 +184,34 @@ Below are instructions for installing the `olm` library for Linux (Debian and Ub
-Minimum required go version is 1.24
+Minimum required go version is written in [go.mod](go.mod)
+
+We recommend not using global go installation but rather installing [gvm](https://github.com/moovweb/gvm) for easier go version management. After installing go, run the following command
 
 ```bash
 ./scripts/build.sh
 ```
+### Troubleshooting
+
+If on macOs you are getting
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)